data/_ui/tooltips.txt

```diff
@@ -1188,6 +1188,9 @@ tip "no fuel?"
 tip "no bays?"
 	`There are insufficient bays available to carry this ship. If it took off, it would be unable to leave this star system.`
 
+tip "inefficient cooling?"
+	`This ship has inefficient cooling. Remove some outfit expansions to improve efficiency.`
```

Comment:
I wouldn't want the tooltip to refer specifically to outfit expansions. While those are the only use of cooling inefficiency in vanilla at the moment, that might not always be the case, and there could always be plugins that make use of the cooling inefficiency attribute.

Reply:
I agree, but felt for now this was good enough. I *do* want to inform novice players about how to fix the situation, and at present, that means removing some Outfit Expansions.

My plan was that plug-ins could override the tool-tip text too.

Reply:
In-game description of the OE says that "Unfortunately, rearranging the interior of your ship in such a fundamental way tends to interfere with its cooling systems", so the player should know that it can cause problems with cooling.
Maybe a way to filter the outfitter to show only items with a selected attribute would help locating the problematic outfits.

Reply:
"Can cause problems" is mild and doesn't imply the scaling function that CoolingEfficiency() applies. The scaling means that you need more than 15 tons of cooling per expansion added. I felt that 0.5 was a good value between "you're going the wrong way" and the red alert that already exists.
Re. cooling UI: Perhaps, and once we have that we can reword the tooltip.

Reply:
> I wouldn't want the tooltip to refer specifically to outfit expansions. While those are the only use of cooling inefficiency in vanilla at the moment, that might not always be the case, and there could always be plugins that make use of the cooling inefficiency attribute.

Is this better? (New commit)